catkin_tools/resultspace.py

```diff
@@ -79,7 +79,7 @@ def get_resultspace_environment(result_space_path, quiet=False, cached=True):
         return {}
 
     # Construct a command list which sources the setup file and prints the env to stdout
-    norc_flags = {'bash': '--norc', 'zsh': '-f'}
+    norc_flags = {'bash': '--norc', 'sh': '', 'zsh': '-f'}
```

Comment:
I guess it's ok to assume that `sh` doesn't source an rc file, though I'm not sure that's true since `sh` can be different things on different OS's. @jbohren what do you think about this?